Dockerfile.client

```diff
@@ -11,7 +17,6 @@ ARG REACT_APP_OKTA_ISSUER
 ARG REACT_APP_OKTA_REDIRECT_URI
 ARG REACT_APP_OKTA_SERVER_ID
 
-RUN yarn install
```

Comment:
What's this about?

Reply:
I moved `RUN yarn install` higher up in the dockerfile (line 6) so that it'll create a layer with just the node dependencies. Those dependencies should change less frequently than the application code, so the change should help with image caching.